src/app/components/Brand/index.jsx

```diff
@@ -4,29 +4,37 @@ import { bool } from 'prop-types';
 import {
   C_POSTBOX,
   C_WHITE,
-  GEL_SPACING_DBL,
+  GEL_SPACING,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_HLF,
 } from '../../lib/constants/styles';
 import {
   layoutGridWrapper,
   layoutGridItem,
   layoutGridItemFullWidth,
 } from '../../lib/layoutGrid';
 
+const SVG_TOP_OFFSET = '1.25rem'; // 20px
+const SVG_BOTTOM_OFFSET = GEL_SPACING_TRPL;
+const BANNER_HEIGHT = '5rem'; // 80px
+
+const SVG_HEIGHT_PX = 24;
+const SVG_HEIGHT = GEL_SPACING_TRPL;
```

Comment:
In this case, for consistency with the Width value, `${SVG_WIDTH_PX / 16}rem`; would make sense, rather than linking it with GEL Spacing, to make it clear in the file where this value comes from.